communitiesuk/funding-service-design-fund-store
# FS-4071: Add contact_us_banner_json to fund-store round model

### Change description

- contact_us_banner_json is a html str which can be custom per round
- For COF, we have a HTML component with the new design for contact page
- For all other funds and their rounds, we have a placeholder empty str
- Add migration script to update the round model with contact_us_banner
- Update the API endpoint retrieving rounds to get contact_us_banner by language

---


### How to test

- After checking out and running/deploying this branch
- You will need to run the `scripts.load_all_fund_rounds` script to populate the database
- Afterwards, using a database client, you can observe the changes have been made to the database
